build/musicbrainz-dev/scripts/fetch-dump.sh

```diff
@@ -98,17 +98,21 @@ then
 	exit 64 # EX_USAGE
 fi
 
+# Keep support for (deprecated) FTP option (which still takes precedence)
```

Comment:
Maybe I've just missed the comment elsewhere but why does this take precedence? Wouldn't it be better to have the base URL as the preferred one and the deprecated one as just a fallback?

Reply:
Actually the commit https://github.com/metabrainz/musicbrainz-docker/pull/245/commits/170055a75b3eab549e290409c33a61a7e9dfed8f is amending the previously merged commit https://github.com/yvanzo/musicbrainz-docker/commit/fddeeaa262c0481fed9ab1bf71e0e7073c5291be whose message explains it all.

I will add a reference to this commit in the new commit message.

Reply:
Oh, ok. I see that the variable is set by default, so that's why it would otherwise override a custom FTP one. Now it makes sense, thanks!

Reply:
I clarified this in message for commit https://github.com/metabrainz/musicbrainz-docker/pull/245/commits/0236211d54d4545faf98e7af723970be507627d7.